client/js/array-util.js

```diff
@@ -1,12 +1,24 @@
+//Returns array from fromNum to toNum that is sequential in order, both increasing and decreasing
 const getRange = function(fromNum, toNum) {
```

Comment:
Set your text editor to use 2-space tabs instead of 4-space tabs; ask me if you have any trouble. This is key for readability, especially when you checked your code into a central repo that other devs will work off of.